README.md

```diff
@@ -36,7 +37,16 @@ Visting http://localhost:8080/ with a web browser will display the application.
 $ make test
 ````
 
+## License
+This project is Copyright (c) 2014-2020 Nimble. It is free software,
+and may be redistributed under the terms specified in the [LICENSE] file.
+
+[LICENSE]: /LICENSE
+
+
```

Comment:
Could we remove extra space here?

Reply:
@junan Fixed https://github.com/Lahphim/go-crawler-challenge/pull/33/commits/299435d346c848437cc801c9f272123f7e4941d8